db/migrate/20181020103501_revoke_variant_overrideswithout_permissions.rb

```diff
@@ -0,0 +1,17 @@
+class RevokeVariantOverrideswithoutPermissions < ActiveRecord::Migration
+  def up
+    # This process was executed when the permission_revoked_at colum was created (see AddPermissionRevokedAtToVariantOverrides)
+    #   It needs to be repeated due to #2739
+    variant_override_hubs = Enterprise.where(id: VariantOverride.all.map(&:hub_id).uniq)
+
+    variant_override_hubs.each do |hub|
+      permitting_producer_ids = hub.relationships_as_child
+        .with_permission(:create_variant_overrides).map(&:parent_id)
```

Comment:
Another good use of `pluck`:
```suggestion
        .with_permission(:create_variant_overrides).pluck(:parent_id)
```